servers/tfserving_proxy/requirements.txt

```diff
@@ -1,3 +1,5 @@
 tensorflow>=1.12.0
 tensorflow-serving-api>=1.10.1
+grpcio==1.32.0 # Required for https://github.com/SeldonIO/seldon-core/issues/2787
```

Comment:
Any link to the tensorflow issue that this fixes? Maybe add to seldon-core issue.

Reply:
It's not really an issue, it's just that they have a hard requirement for grpcio==0.32.0, whereas seldon-core package installs 1.34.0 on the base redhat image

Reply:
So why not fix python package to 0.32.0 ?

Reply:
Ok actually I've added https://github.com/tensorflow/tensorflow/issues/45785 to see if they can actually update it, but let's see, I do expect them to just say "we'll need to stick with 1.32.x" for Y reasons